core/src/main/scala/cats/Reducible.scala

```diff
@@ -112,6 +112,12 @@ import simulacrum.typeclass
       val F = self
       val G = Reducible[G]
     }
+
+  def minimum[A](fa: F[A])(implicit A: Order[A]): A =
```

Comment:
@johnynek yes that's what I mean, but note that we already have syntax enrichment for `Reducible`, and it's generated by Simulacrum so it will automatically inherit the name of the method on the type class.